cost/azure/instances_log_analytics_utilization/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v2.8
+
+- Added 404 ignore to azure instances datasource
```

Comment:
This isn't quite what you changes.  It's ignoring log analytics workspaces that don't exists.  Would you update this to give the customer a better idea of what was changes.  thx

Reply:
Updated CHANGELOG to reflect the edit above